packages/editor-ui/src/constants.ts

```diff
@@ -541,21 +545,25 @@ export const enum WORKFLOW_MENU_ACTIONS {
 /**
  * Enterprise edition
  */
-export const enum EnterpriseEditionFeature {
-	AdvancedExecutionFilters = 'advancedExecutionFilters',
-	Sharing = 'sharing',
-	Ldap = 'ldap',
-	LogStreaming = 'logStreaming',
-	Variables = 'variables',
-	Saml = 'saml',
-	SourceControl = 'sourceControl',
-	ExternalSecrets = 'externalSecrets',
-	AuditLogs = 'auditLogs',
-	DebugInEditor = 'debugInEditor',
-	WorkflowHistory = 'workflowHistory',
-	WorkerView = 'workerView',
-	AdvancedPermissions = 'advancedPermissions',
-}
+export const EnterpriseEditionFeature: Record<
+	EnterpriseEditionFeatureKeys,
+	EnterpriseEditionFeatureValues
+> = {
```

Comment:
Can you please rename these two to be singular, to avoid confusion on whether the type represents a single value or an array of values?